src/OpenTelemetry/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* The opt-in overflow attribute feature which can be enabled by setting the
+  environment variable `OTEL_DOTNET_EXPERIMENTAL_METRICS_EMIT_OVERFLOW_ATTRIBUTE`
+  to `true` is now enabled by default and supported in stable builds.
+  ([#5909](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5909))
```

Comment:
please clarify if the ENV variable can still be used to turn this capability off? I think with this PR, SDK stop recognizing this env variable totally.

Reply:
Updated Cardinality Limit Behavior in OpenTelemetry Metrics SDK
**Previous Behavior:**  
By default, when the cardinality limit was reached, measurements were dropped, and an internal log was emitted the first time this occurred. Users could opt-in to `OTEL_DOTNET_EXPERIMENTAL_METRICS_EMIT_OVERFLOW_ATTRIBUTE=true` to enable an alternative behavior. With this setting, the SDK would report an overflow attribute (`otel.metric.overflow = true`) instead of dropping measurements, ensuring metric accuracy. No internal log was emitted in this case.

**New Behavior:**  
The SDK now always adds the overflow attribute (`otel.metric.overflow = true`) to metrics when the cardinality limit is reached. The previous approach of dropping measurements has been removed. No internal logs are emitted when the limit is hit, as the `otel.metric.overflow` attribute indicates when a cardinality cap has occurred.

The default cardinality limit remains 2000 per metric. To adjust this globally, use the `SetMaxMetricPointsPerStreams()` API. However, we recommend setting the limit on a per-metric basis for finer control. For details on customizing limits per metric, refer to [changing cardinality limit](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/customizing-the-sdk/README.md#changing-the-cardinality-limit-for-a-metric).

There is *NO* ability to revert to old behavior.


^ See if the above is better. @CodeBlanch @alanwest please help polish the messaging in the changelog

Reply:
This variable will be gone. I'll update the wording to make it more clear.

Reply:
Updated.

Reply:
I still suggest to keep the before/after style in the changelog, as that will make it easier for end users to know what changed.

Reply:
I've added the details here with some minor edits. Can you review again?